cpu/arm7_common/Makefile.features

```diff
@@ -6,7 +6,11 @@ FEATURES_PROVIDED += arch_arm
 FEATURES_PROVIDED += arch_arm7
 FEATURES_PROVIDED += cpp
 FEATURES_PROVIDED += libstdcpp
+FEATURES_PROVIDED += newlib
 FEATURES_PROVIDED += periph_pm
 FEATURES_PROVIDED += picolibc
 FEATURES_PROVIDED += puf_sram
 FEATURES_PROVIDED += ssp
+
+FEATURES_CONFLICT += picolibc:newlib
```

Comment:
I think this should be moved in the main Makefile.features, to avoid duplicated code.

Reply:
If I do so then it would show up for all arch though.

Reply:
But if you prefer it that way I can change it, let me know.

Reply:
> it would show up for all arch though

Indeed and that would be normal I guess since both cannot be used at the same time. So the next time picolibc is added to a new arch (could be the case on esp32), there will be no risk to forget to add the libc feature conflict.

Reply:
Addressed